src/components/constraints/ConstraintsPanel.svelte

```diff
@@ -76,19 +80,33 @@
     } else {
       endTime = '';
     }
+    hasSpecEditPermission = featurePermissions.constraintsPlanSpec.canUpdate(user, $plan) && !$planReadOnly;
+
+    editPermissionError = $planReadOnly
```

Comment:
Seeing this error when I should be seeing the other error about the last invocation deletion? I followed test instructions in the PR (as aerie admin), duplicated the constraint, deleted the second constraint, and tried to delete the first constraint.
<img width="446" alt="image" src="https://github.com/user-attachments/assets/36cc8341-29a6-4eae-b1d8-1480ef9109d6" />